lib/ar_query_matchers.rb

```diff
@@ -7,6 +7,20 @@
 
 module ArQueryMatchers
   module ArQueryMatchers
+    class Utility
+      def self.clean_expected(expected)
```

Comment:
perhaps a more concrete name:
```suggestion
      def self.remove_superfluous_expectations(expected)
```

Reply:
Cool, done